Snakefile

```diff
@@ -74,6 +74,15 @@ def read_skip_steps_file():
                 print(exc)
         return skip_steps
 
+def read_run_deconv_file():
+    if 'run_deconv_file' in config:
+        global run_deconv
+        with open(config['run_deconv_file'], 'r') as stream:
+            try:
+                run_deconv=yaml.safe_load(stream)
+            except yaml.YAMLError as exc:
+                print(exc)
```

Comment:
Shouldnt it exit if error in yaml file

Reply:
Thanks, added here [2485975](https://github.com/ebi-gene-expression-group/bulk-recalculations/pull/65/commits/2485975148e7b32abfabc8122f152397f17c2f5d)